qa/L0_server_launch_modes/test_config_generator.py

```diff
@@ -102,6 +102,17 @@ def set_triton_server_path(self, launch_mode, config):
         if launch_mode != 'local':
             config['triton_server_path'] = '/path/to/nowhere'
 
+    def set_triton_install_path(self, launch_mode, config):
+        """
+        Helper function to set the 'triton_install_path' to something bogus
+            for all configuration except 'c_api'
+        This ensures only the 'c_api' launch mode uses 'triton_install_path'.
+        """
+        config.pop('triton_install_path', None)
+
+        if launch_mode != 'c_api':
+            config['triton_install_path'] = '/path/to/nowhere'
+
```

Comment:
Wondering if I should keep `set_triton_server_path` and `set_triton_install_path` separated like this or combine them.

Reply:
I think this is fine to keep separate.  There is almost no shared code